### DISCUSSION TITLE
Aurora Serverless v2

### DISCUSSION THREAD

**Discussion by andrewb85** [gravitational/teleport#13036] [General]
AWS Aurora serverless v2 now supports IAM authentication.  I have this working in teleport 9.0.4 with postgres hosted on Aurora Serverless V2.  I have been hesitant to upgrade to newer versions of teleport because I remember reading newer version of teleport filter out aurora serverless.   Is there a way to upgrade to a newer version and not have this filtering take place or should I just wait for a new version to be released that support Aurora Serverless

**Comment by webvictim** [comment]
Thanks - we've already raised an issue to track this: https://github.com/gravitational/teleport/issues/13018

**Comment by greedy52** [comment]
v9.0.4 already has this filter in effect, but it only filters out Serverless v1. It is safe to upgrade. I've tested Serverless v2 on `master` and it works fine. We will update our documentation to reflect that v2 is supported.
